app/models/storage_provider.rb

```diff
@@ -33,7 +33,12 @@ def gcp_storage_uploader
     @gcp_storage_uploader ||= Atc::Gcp::StorageUploader.new(GCP_STORAGE_CLIENT, self.container_name)
   end
 
-  def store_gcp(pending_transfer, stored_object_key, metadata:, **_args)
+  def store_gcp(pending_transfer, stored_object_key, metadata:, tags: nil)
```

Comment:
an alternative would be to check to see if the `**_args` was empty or not - this would afford you some flexibility about signatures down the line. If you did that, you should also rename `_args` to `args`, since it would be used!

Reply:
I actually started off doing that, but found myself needing to then update the `peform_transfer` signature to `perform_transfer(pending_transfer, stored_object_key, **args)` and then passing `**args` to both the `store_aws` and `store_gcp` methods.  One other option would have been to keep the `perform_transfer` signature the same and just not call store_gcp with `tags: nil`, but then the invalid kwarg validation would have had to move up to a higher level than the `store_gcp` method, and check storage_type.

I'm thinking that because `tags` is the only conditional keyword arg for now, maybe it's easiest to go with the current code and consider a more general `**args`-type update in a future PR?  (And we _might_ even consider removing `tags` altogether if it ends up not being applicable to any of our use cases.)  Does that sound okay?

Reply:
I'll merge since you approved the PR, but glad to talk this over later if you want to!